hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/util/WrappingProcessorMetaSupplier.java

```diff
@@ -56,4 +56,9 @@ public int preferredLocalParallelism() {
     public void init(@Nonnull Context context) {
         wrapped.init(context);
     }
+
+    @Override
+    public void complete(Throwable error) {
```

Comment:
seems there are missing tests for calling of the complete() methods for the wrapping processors.

Reply:
i was thinking of having a test that will check that all methods of PS/PMS are overridden in WPS/WPMS.

Reply:
Yes, we have a similar test which use Powermock for this for IStreamMap/IStreamList.. (`DecoratorTest`)